### ISSUE TITLE
no-implicit-coercion: Show the only the first line of violation

### ISSUE THREAD

**Issue by moylop260** [eslint/eslint#6810]
**What version of ESLint are you using?**
eslint --version
v3.2.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

``` json
{
  "rules": {
    "no-implicit-coercion": "error"
  },
}
```

**What did you do? Please include the actual source code causing the issue.**
I have created a `file.js` with the following code:

``` js
+function ($) {
  'use strict';
  var var_1 = "value1";
  var var_2 = "value2";
  // many lines here
}
```

Run command:
`eslint --format=unix file.js`

**What did you expect to happen?**
Show 

``` bash
file.js:1:1: use `Number(function ($) ...` instead. [Error/no-implicit-coercion]

1 problem
```

**What actually happened? Please include the actual, raw output from ESLint.**
Show 

``` bash
file.js:1:1: use `Number(function ($) {
  'use strict';
  var var_1 = "value1";
  var var_1 = "value1";
  // many lines here
})` instead. [Error/no-implicit-coercion]

1 problem
```

**Comment by vitorbal** [comment]
Hi @moylop260, thanks for reaching out! Just to be clear, are you proposing that we limit the amount of characters that we show in the error message of this rule?

**Comment by eslintbot** [comment]
Hi @moylop260, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by moylop260** [comment]
Thanks for reply

@vitorbal
Yes, I have a file with many lines and the output is hijacked by this check because are showed all lines.
The most important information is just the fisrt word of the refactoring proposal.

IMHO should be showed just the first word without the full file.

**Comment by platinumazure** [comment]
Thanks @moylop260 for the clarification. I hope it's okay that I have changed the issue title for clarity.

At this point, I'm treating the current behavior as "designed", and I see this as an enhancement request to report a shorter message. (I think we can start with just reporting the first line, and see what happens from there. Sound okay?)

I'm :+1: to this change.

**Comment by moylop260** [comment]
@platinumazure
Just reporting the first line LGTM
Thaks

**Comment by kaicataldo** [comment]
I'm closing this issue because it looks like consensus couldn't be reached. While we wish we could accommodate all requests, we have limited resources and do need to prioritize. We've found that issues failing to reach consensus after 21 days tend to never reach consensus, and as a team have decided to close such issues. This doesn't mean the idea isn't interesting or valuable, only that it's not something the team can commit to at the moment.
